### ISSUE TITLE
[Feature request] Package binary also for arm darwin M1/M2 chips

### ISSUE THREAD

**Issue by bennesp** [solokeys/solo2-cli#78]
I can see in packages three different packages for darwin, linux and windows, all x86_64

It would be great to be able to use this CLI natively also on arm chips like the M1/M2 ones.

**Comment by tehabe** [comment]
Yeah, a universal build for macOS supporting x86_64 and arm64 would be awesome. The x86_64 build works thanks to Rosetta 2 but who knows how long Apple will support it.
